### ISSUE TITLE
Why use SPFx 1.8 with TypeScript 2.7?

### ISSUE THREAD

**Issue by BigEaseGueldi** [SharePoint/sp-dev-docs#3618]
#### Category
- [X] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
I was trying to use Fabric UI Icons and other components with SPFx 1.8 - **without** using the loadLegacyFabricCss flag. 

#### Observed Behavior
Once you add something, e.g.
**_import { Icon } from 'office-ui-fabric-react/lib/Icon';_**
to any component, you will see a "good old friend" (frankly speaking there are "some" issues around this ;-)):

![grafik](https://user-images.githubusercontent.com/17311219/54477984-9b5f4e80-480d-11e9-9d26-3590e0af2613.png)

#### Steps to Reproduce

see above

So the question would be why you whould use TS 2.7, when you have to switch to 3.x to use Fabric UI properly. Are there any disadvantages when switching to TS 3.x?

Thanks
Dirk

**Comment by PooLP** [comment]
Hi,
I had this error after update a Web Part to 1.8, I have solved this with this excellent post by @estruyf :
https://www.eliostruyf.com/using-the-full-potential-of-react-in-sharepoint-framework-1-8-0/

**Comment by BigEaseGueldi** [comment]
Ok, let me rephrase that:
Given the fact, that many (most?) SPFx solutions use Fabric UI styles / components, why doesn't the standard package allow for it? Can't imagine that using e.g. TS 3.3 only brings advantages, otherwise it would have been there in the standard.

**Comment by patmill** [comment]
Different version of the Office UI Fabric React work with different compilers. Internally, we are using "office-ui-fabric-react": "6.143.0", with the 2.x compilers.

**Comment by Paul-Colucci** [comment]
ui fabric react 6.143.0 still doesn't compile for me?  Are there other work arounds? I just want to compile a simple webpart that uses fabric.  It is insane that they would release this with mis-matched versions of libraries and tools all over the place.

**Comment by VesaJuvonen** [comment]
This is a versioning conflict in usage of the Fabric which requires a different TypeScript version. We will be releasing guidance for this as part of the upcoming 1.8.2 version.

**Comment by BigEaseGueldi** [comment]
Closing this as it's been resolved with 1.8.2.
